v1alpha1/meshmodel.proto

```diff
@@ -174,30 +174,33 @@ message Ingress {
 
 // ServiceInstance is the runnable entity of a Mesh Service.
 message ServiceInstance {
+	// registryName is the name of registry.
```

Comment:
If this requires frontward compatibility, this field should be added as the last field in PB which is the eighth field. 
Ignore this suggestion if no such require needed.

Reply:
We will publish a new version of Easegress, EaseMesh, and EaseMeshAPI including this or and https://github.com/megaease/easegress/pull/62 . The RESTful API adjustment is incompatible, but it must be adjusted to the right way. So we do it at one time.

Reply:
Ok, I got it.